### ISSUE TITLE
Files Not Found on CentOS

### ISSUE THREAD

**Issue by EarthCitizen** [marcules/karma-remap-istanbul#16]
We have a the configuration below which works fine on OS X using karma-remap-istanbul 0.1.0:

```
    remapIstanbulReporter: {
      src: 'reports/coverage.json',
      reports: {
        lcovonly: 'reports/ts-ut.lcov',
        html: 'reports/html'
      },
      timeoutNotCreated: 5000,
      timeoutNoMoreFiles: 1000
    },
```

But on CentOS it fails with 'Could not find any specified files, exiting without doing anything'. I have tried CentOS 7 and 6.5. Same result.

I put in an issue with chokidar:

https://github.com/paulmillr/chokidar/issues/514#issuecomment-235973032

But the owner thinks that the settings being used by karma-remap-istanbul might be the issue.

**Comment by EarthCitizen** [comment]
I just tried on an Ubuntu 16 box and got the same result.

**Comment by delasteve** [comment]
Should be fixed in `0.2.1`.
